ios/FluentUI/Bottom Commanding/BottomCommandingController.swift

```diff
@@ -546,7 +568,16 @@ extension BottomCommandingController: CommandingItemDelegate {
     }
 
     func commandingItem(_ item: CommandingItem, didChangeToggleableTo value: Bool) {
-        reloadView(from: item)
+        guard let binding = itemToBindingMap[item] else {
+            return
+        }
+
+        // We need a UITableView.reloadRows call here because item.isToggleable changes the cell type
```

Comment:
should we really support this scenario? who would do this? maybe togglable isn't something that is mutable?

Reply:
Keeping all the props mutable is good for OfficeSpace integration, because it sets user interface element properties one by one during the binding process. You're right, it doesn't really make much sense in regular use, but I don't see the harm in keeping it mutable.

Reply:
Well OfficeSpace doesn't have to. The general design is immutable properties get set by the factory and mutable properties are set during binding. We technically could eliminate the need for mutability in our usage if we don't think this is a realistic or intended scenario

Reply:
Ah, I missed that. The factory would know whether it's creating an FSBooleanChoice so it can just init the item with the correct isToggleable value. Cool! I'll remove the mutability then.